install.py

```diff
@@ -7,6 +6,21 @@
 from pathlib import Path
 from typing import Tuple, Optional
 
+try:
+    import launch
+except ModuleNotFoundError:
+    print("ControlNet cannot resolve the 'launch' module from the webui. Attempting to add the webui path to sys.path.")
+
+    # Add webui to interpreter path
+    webui_path = Path(__file__).parent.parent.parent.absolute().resolve()
```

Comment:
this is also not a valid fix
extension directory may not be under webui root
it can be moved by [`--data-dir`](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings)

Reply:
that's true,
I working with stable diffusion environment for only 4 days and for only a small project.
I don't know how it is work from scratch.
can I ask for help from both of you? @w-e-w @huchenlei